api.js

```diff
@@ -195,7 +195,7 @@ function handlePaths(files) {
 		files = [
 			'test.js',
 			'test-*.js',
-			'test/*.js'
```

Comment:
There's a good reason the inconsistency, though. It's only safe to recurse the `test` folder. We could of course only recurse the test folder, but most tiny modules (like most of mine) have a single `test.js` file in root or a couple of `test-*.js`, so it's useful to support that too.

> **/test.js

This would match https://github.com/sindresorhus/ava/blob/master/lib/test.js which confirms my above argument.

> **/*.spec.js **/*.test.js

I'd prefer to wait until someone actually asks for it. I feel it's better to keep your tests separate from the code in a dedicated `test` folder.